client/state/document/reducer.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SET_DOCUMENT_DESCRIPTION,
+	SET_DOCUMENT_TITLE,
+} from 'state/action-types';
+
+const INITIAL_STATE = {
+	title: '',
+	description: '',
+};
+
+export function title( state = '', action ) {
+	switch ( action.type ) {
+		case SET_DOCUMENT_TITLE:
+			return action.title;
+	}
+
+	return state;
+}
+
+export function description( state = '', action ) {
```

Comment:
Description itself is a `<meta>` tag when rendered to the page. How do you plan to reconcile this with our other `<meta>` needs?

In the course of looking this up, I realized that canonical URL ([as required by themes](https://github.com/Automattic/wp-calypso/blob/539f4e8e93b088517b87464b5779ad2eaf43fc31/client/my-sites/themes/head.jsx#L27-L43)) is assigned with a `<link>` tag, not a `<meta>` tag, so we'll need to consider this as a requirement as well.

Given the latter discovery, I'm not sure we currently do have other `<meta>` requirements, but I might imagine we'd eventually want to, for example, add [Open Graph tags](http://ogp.me/).